installer/install.iss

```diff
@@ -40,8 +40,15 @@ OutputDir={#GetEnv('USERPROFILE')}
 SolidCompression=yes
 #ifdef SOURCE_DIR
 SourceDir={#SOURCE_DIR}
+#define FILE_VERSION GetFileVersion(SOURCE_DIR+"\"+MINGW_BITNESS+"\bin\git.exe")
+#define PROD_VERSION GetStringFileInfo(SOURCE_DIR+"\"+MINGW_BITNESS+"\bin\git.exe", \
+			"ProductVersion")
 #else
 SourceDir={#SourcePath}\..\..\..\..
+#define FILE_VERSION GetFileVersion(SourcePath+"\..\..\..\..\" \
+			+MINGW_BITNESS+"\bin\git.exe")
+#define PROD_VERSION GetStringFileInfo(SourcePath+"\..\..\..\..\" \
+			+MINGW_BITNESS+"\bin\git.exe", "ProductVersion")
```

Comment:
Why not use `ExpandConstant('{#SourceDir}')` (or something similar to make use of `SourceDir`) for both arms, making them identical, i.e. you can move the definition of `FILE_VERSION` and `PROD_VERSION` out of the conditional?